infrastructure/terraform/components/reporting/sfn_state_machine_athena.tf

```diff
@@ -169,4 +175,25 @@ data "aws_iam_policy_document" "sfn_athena" {
     ]
   }
 
+  statement {
+    sid    = "AllowCloudwatchLogging"
+    effect = "Allow"
+
+    actions = [
+      "logs:CreateLogDelivery",
+      "logs:CreateLogStream",
+      "logs:GetLogDelivery",
+      "logs:UpdateLogDelivery",
+      "logs:DeleteLogDelivery",
+      "logs:ListLogDeliveries",
+      "logs:PutLogEvents",
+      "logs:PutResourcePolicy",
+      "logs:DescribeResourcePolicies",
+      "logs:DescribeLogGroups"
+    ]
+
+    resources = [
+      "*", ## https://docs.aws.amazon.com/step-functions/latest/dg/cw-logs.html
```

Comment:
Can this be scoped better please?

Reply:
The link in the comment explains it better than I can. I did initially have it scoped to the log group itself but it doesn't work.

Reply:
I've addressed this to some extent by splitting out into two separate statements, scoping where I can and adding a conditional to ensure the requestingtag is for the same environment.